### PR TITLE
Edits to PGD PR5384

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/pgd/5/scaling.mdx [hunk 1]
```diff
@@ -23,10 +23,10 @@ function to create or alter the definition of automatic range partitioning for a
 no definition exists, it's created. Otherwise, later executions will alter the
 definition.
 
-PGD Autopartition in PGD 5 currently locks the actual table while performing 
+PGD AutoPartition in PGD 5 currently locks the actual table while performing 
 new partition maintenance operations.
 
-An ERROR is raised if the table isn't RANGE partitioned or a multi-column
+An error is raised if the table isn't range partitioned or a multi-column
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/scaling.mdx:29]: Must range be all caps? I changed it here and below...I think it's probably okay like this?

Comment (djw-m) [reply product_docs/docs/pgd/5/scaling.mdx]: Should be ok. This is an old doc.

File: product_docs/docs/pgd/5/scaling.mdx [hunk 2]
```diff
@@ -174,12 +174,12 @@ function to find the partition for the given partition key value. If partition
 to hold that value doesn't exist, then the function returns NULL. Otherwise Oid
 of the partition is returned.
 
-## Enabling or disabling AutoPartitioning
+## Enabling or disabling autopartitioning
```
[the remark this answers is not in the document]
Comment (djw-m) [reply product_docs/docs/pgd/5/scaling.mdx:177]: I think it was to be a badged feature but less so currently.

**Review by djw-m — APPROVED** [review]
LGTM

**Pull request merged**